### PR TITLE
[tiptap] plugins toolbar should create by settings

### PR DESCRIPTION
**Pull request by danarish4**

### CODE REVIEW

File: packages/ricos-editor/web/src/tiptap/RicosToolbars.tsx [hunk 1]
```diff
@@ -263,6 +269,43 @@ class RicosToolbars extends React.Component<
     return null;
   }
 
+  renderPluginsToolbar = (finaltoolbarSettings: ToolbarSettingsFunctions[]) => {
+    const { content } = this.props;
+    const toolbarType = TOOLBARS.INLINE;
```
Comment (noamtwix) [inline packages/ricos-editor/web/src/tiptap/RicosToolbars.tsx:274]: are you sure it's `TOOLBARS.INLINE`?
there is also `TOOLBARS.PLUGIN`, but we need to make sure what is the correct one.

Comment (danarish4) [inline packages/ricos-editor/web/src/tiptap/RicosToolbars.tsx:274]:
```suggestion
    const toolbarType = TOOLBARS.PLUGIN;
```

File: packages/ricos-editor/web/src/tiptap/RicosToolbars.tsx [hunk 2]
```diff
@@ -263,6 +269,43 @@ class RicosToolbars extends React.Component<
     return null;
   }
 
+  renderPluginsToolbar = (finaltoolbarSettings: ToolbarSettingsFunctions[]) => {
+    const { content } = this.props;
+    const toolbarType = TOOLBARS.PLUGIN;
+    const toolbarConfig = this.getToolbarConfig(finaltoolbarSettings, toolbarType);
+
+    if (toolbarConfig?.shouldCreate?.()) {
```
Comment (maimonav) [inline packages/ricos-editor/web/src/tiptap/RicosToolbars.tsx:277]: I've pushed a fix, for the shouldCreate usage in this file, for [example](https://github.com/wix/ricos/commit/c554e029812365a70c68048e66d318122dfa3c27#diff-dfb694da2ec399c063ad2a8b323725c619ea37a03efd6f091fefa5a732722c1cR168)

Comment (maimonav) [reply packages/ricos-editor/web/src/tiptap/RicosToolbars.tsx:277]: Please change the shouldCreate usage accordingly in all cases.

**Pull request merged**